src/components/molecules/DataWrapper.tsx

```diff
@@ -1,22 +1,29 @@
 import React, { ReactNode, useEffect, useState } from 'react';
-import { ApolloError, DocumentNode, useQuery } from '@apollo/client';
+import { DocumentNode, useQuery } from '@apollo/client';
 import { TimeEntry } from '../atoms/ThresholdChart';
 import { formatMetData } from '../../utils/formatMetData';
 
 export type DataWrapperProps = {
     query: DocumentNode;
-    children: (args: { data: TimeEntry[] | undefined; loading: boolean; error: ApolloError | undefined }) => ReactNode;
+    children: (args: { data: TimeEntry[] }) => ReactNode;
 };
 
 const DataWrapper: React.FC<DataWrapperProps> = ({ children, query }) => {
-    const [formattedData, setFormattedData] = useState<TimeEntry[] | undefined>(undefined);
+    const [formattedData, setFormattedData] = useState<TimeEntry[]>([]);
     const { loading, error, data } = useQuery(query);
 
     useEffect(() => {
         if (data) setFormattedData(formatMetData(data));
     }, [data]);
 
-    return <>{children({ data: formattedData, loading, error })}</>;
+    if (loading) {
+        return <p>Loading...</p>; //Change to loading atom
+    }
+    if (error) {
+        return <p>Error :/</p>; //Change to error atom
```

Comment:
Might be helpful to render `error.message` for the time being while a proper atom is being constructed.